pkg/cmd/openshift-tests/monitor/timeline/timeline_command.go

```diff
@@ -247,7 +247,7 @@ func renderHTML(events monitorapi.Intervals) ([]byte, error) {
 		return nil, err
 
 	}
-	e2eChartTemplate := testdata.MustAsset("e2echart/non-spyglass-e2e-chart-template.html")
+	e2eChartTemplate := testdata.MustAsset("e2echart/e2e-chart-template.html")
```

Comment:
This command is not used anywhere other than by devs as far as I can see. I switched it to default to the template we look at most often.